components/playlist/media_detector_component_manager.cc

```diff
@@ -241,4 +246,17 @@ void MediaDetectorComponentManager::SetUseLocalScriptForTesting() {
   OnGetScript(kScript);
 }
 
+bool MediaDetectorComponentManager::ShouldHideMediaSrcAPI(
+    const GURL& url) const {
+  return base::ranges::any_of(sites_to_hide_media_src_api_,
+                              [&url](const auto& site_to_hide) {
+                                return site_to_hide == net::SchemefulSite(url);
```

Comment:
`net::SchemefulSite(url)` is created each time this lambda is called. pls move it from the lambda.

Reply:
Good point! done.